src/plugins/tags/tags.command.ts

```diff
@@ -0,0 +1,108 @@
+import { ChatInputCommandInteraction, inlineCode } from "discord.js";
+import {SlashCommandBuilder, SlashCommandScope,} from "../../builders/SlashCommandBuilder";
+import { useChatCommand } from "../../hooks/useChatCommand";
+import { Tag } from "./Tag.model";
+
+const builder = new SlashCommandBuilder()
+.setName("tags")
+.setDescription("Manages tags.")
+.addSubcommand((subcommand) => 
+    subcommand
+        .setName("list")
+        .setDescription("Lists all tags for this server.")
+    )
+.addSubcommand((subcommand) => 
+    subcommand
+        .setName("create")
+        .setDescription("Creates a tag with the given name and content.")
+        .addStringOption((option) => 
+            option.setName("name").setDescription("Name for the tag to create").setRequired(true)
+        )
+        .addStringOption((option) => 
+        option.setName("content").setDescription("Content for the tag to create").setRequired(true)
+        )
+    )
+.addSubcommand((subcommand) => 
+subcommand
+    .setName("delete")
+    .setDescription("Deletes a tag.")
+    .addStringOption((option) => 
+        option.setName("name").setDescription("Name for the tag to delete").setRequired(true)
+    )
+)
+.addSubcommandGroup((group) => 
+    group
+    .setName("edit")
+    .setDescription("Edits a tag.")
+    .addSubcommand((subcommand) => 
+        subcommand
+        .setName("name")
+        .setDescription("Edits a tag's name")
+        .addStringOption((option) =>
+            option.setName("name").setDescription("Name for the tag").setRequired(true)        
+        )
+        .addStringOption((option) =>
+        option.setName("new_name").setDescription("New name for the tag").setRequired(true)        
+    )
+    )
+    .addSubcommand((subcommand) => 
+    subcommand
+    .setName("content")
+    .setDescription("Edits a tag's content.")
+    .addStringOption((option) =>
+        option.setName("name").setDescription("Name for the tag").setRequired(true)        
+    )
+    .addStringOption((option) =>
+    option.setName("new_content").setDescription("New content for the tag").setRequired(true)        
+)
+)
+)
+
+useChatCommand(builder as SlashCommandBuilder, async (interaction: ChatInputCommandInteraction) => {
+    const subcommand = interaction.options.getSubcommand();
+    
+    if (!interaction.guild) throw new Error(`This command can only be ran in guilds.`)
+    const guildId = interaction.guild.id
+    
+    if (subcommand === "list") {
+        const tags = await Tag.find({guild: guildId});
+        if (tags.length === 0) {
+            return `No tags were found for this server.`
```

Comment:
"There are no tags in this server!" would probably be a better message for the user as this one implies there could be some the bot didn't find